lib/IAMClient.js

```diff
@@ -202,14 +201,25 @@ class IAMClient {
         assert(algo === 'sha1' || algo === 'sha256',
                'options.algo must be sha1 or sha256');
 
-        const data = { additionaldata: { stringToSign: string,
-                                         signatureFromRequest: signature,
-                                         hashAlgorithm: algo,
-                                         accessKey } };
-        if (options.reqUid !== undefined) {
-            data[requestUidKey] = options.reqUid;
-        }
-        this.request('GET', '/auth/v2', callback, data);
+        const data = {
```

Comment:
Cool.  I can use this object to send info along for the authorization (policy evaluation) request as well.